spec/requests/api/v1/users/account/profiles_spec.rb

```diff
@@ -35,5 +35,14 @@
         include_examples 'renders uri query errors'
       end
     end
+
+    context 'N+1', :n_plus_one do # rubocop:disable RSpec/ContextWording
```

Comment:
why disabling here?
why context - not 'describe'?

Reply:
i just followed examples, you right better will be `describe`